cli/mender-artifact/main.go

```diff
@@ -212,6 +212,20 @@ func run() error {
 		Action:      Copy,
 	}
 
+	remove := cli.Command{
+		Name:        "rm",
+		Usage:       "rm [artifact|sdimg|uefiimg]:<filepath>",
+		Description: "Removes the given file or directory from an Artifact or sdimg.",
```

Comment:
To be consistent, can we use `Artifact` also in the other help texts? copy, cat, install, ...

Reply:
Changed it up for everything except cat, where the usage text has it covered: `Copy from or into an artifact, or sdimg where either the <src> or <dst> has to be of the form [artifact|sdimg]:<filepath>`
Otherwise the usage line will be very long.